NotificationServiceExtension/NotificationService.swift

```diff
@@ -2,57 +2,76 @@
 import UserNotifications
 import WMF
 
-class FakeLangProvider: WMFPreferredLanguageInfoProvider {
-    func getPreferredContentLanguageCodes(_ completion: @escaping ([String]) -> Void) {
-        completion(["en"])
-    }
-
-    func getPreferredLanguageCodes(_ completion: @escaping ([String]) -> Void) {
-        completion(["en"])
-    }
-
-
-}
-
 class NotificationService: UNNotificationServiceExtension {
 
     var contentHandler: ((UNNotificationContent) -> Void)?
     var bestAttemptContent: UNMutableNotificationContent?
-    lazy var remoteNotificationsController: RemoteNotificationsController = {
+    
+    private lazy var apiController: RemoteNotificationsAPIController = {
         let configuration = Configuration.current
         let session = Session(configuration: configuration)
-        let fakeProvider = FakeLangProvider()
-        let controller = RemoteNotificationsController(session: session, configuration: configuration, preferredLanguageCodesProvider: fakeProvider)
+        let controller = RemoteNotificationsAPIController(session: session, configuration: configuration)
         return controller
     }()
+    private let sharedCache = SharedContainerCache<PushNotificationsCache>.init(pathComponent: .pushNotificationsCache, defaultCache: { PushNotificationsCache(settings: .default, notifications: []) })
+    
+    private let fallbackPushContent = WMFLocalizedString("notifications-push-fallback-body-text", value: "New activity on Wikipedia", comment: "Fallback body content of a push notification whose content cannot be determined. Could be either due multiple notifications represented or errors.")
 
     override func didReceive(_ request: UNNotificationRequest, withContentHandler contentHandler: @escaping (UNNotificationContent) -> Void) {
         self.contentHandler = contentHandler
         
-        //TODO: We will modify content in all error cases to "New activity on Wikipedia" (see: https://phabricator.wikimedia.org/T288773 > Push Notification Content section)
         guard let bestAttemptContent = request.content.mutableCopy() as? UNMutableNotificationContent else {
             contentHandler(request.content)
             return
         }
         
+        bestAttemptContent.body = fallbackPushContent
```

Comment:
push this to an `else` statement below so that it's clearer how this gets built in all scenarios?

Reply:
Done!